DeleteUserDataActivity/utils.ts

```diff
@@ -106,3 +106,28 @@ export const saveDataToBlob = <T>(
     _ => data
   );
 };
+
+export function flattenAsyncIterator<T>(
```

Comment:
can we import the one from io-fn-commons once merged?

Reply:
Yep!